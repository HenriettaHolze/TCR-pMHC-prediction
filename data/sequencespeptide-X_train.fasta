>peptide-0
FLYALALLL
>peptide-1
GILGFVFTL
>peptide-2
GLCTLVAML
>peptide-3
ILKEPVHGV
>peptide-4
IMDQVPFSV
>peptide-5
KLQCVDLHV
>peptide-6
KTWGQYWQV
>peptide-7
KVAELVHFL
>peptide-8
KVLEYVIKV
>peptide-9
LLFGYPVYV
>peptide-10
MLDLQPETT
>peptide-11
NLVPMVATV
>peptide-12
RMFPNAPYL
>peptide-13
RTLNAWVKV
>peptide-14
SLFNTVATL
>peptide-15
SLLMWITQV
>peptide-16
YLLEMLWRL
