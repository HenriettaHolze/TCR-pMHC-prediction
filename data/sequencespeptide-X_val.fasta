>peptide-0
CLGGLLTMV
>peptide-1
FLYALALLL
>peptide-2
GILGFVFTL
>peptide-3
GLCTLVAML
>peptide-4
IMDQVPFSV
>peptide-5
KTWGQYWQV
>peptide-6
LLFGYPVYV
>peptide-7
MLDLQPETT
>peptide-8
NLVPMVATV
>peptide-9
RMFPNAPYL
>peptide-10
RTLNAWVKV
>peptide-11
SLFNTVATL
>peptide-12
SLLMWITQV
>peptide-13
YLLEMLWRL
