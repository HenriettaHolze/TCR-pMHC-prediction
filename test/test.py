#!/usr/bin/env python
# coding: utf-8
# Imports
import matplotlib.pyplot as plt
import numpy as np
import pandas as pd
import torch
import torch.nn as nn  # All neural network modules, nn.Linear, nn.Conv2d, BatchNorm, Loss functions
import torch.nn.functional as F  # All functions that don't have any parameters
import torch.optim as optim  # For all Optimization algorithms, SGD, Adam, etc.
from sklearn import metrics
from sklearn.metrics import accuracy_score, confusion_matrix, f1_score, roc_curve, matthews_corrcoef
# from google.colab import drive
import os
import torch
import torch.utils.data


device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
print("Using device (CPU/GPU):", device)