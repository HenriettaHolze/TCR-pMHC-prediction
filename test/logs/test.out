Running script...
Using device (CPU/GPU): cuda

------------------------------------------------------------
Sender: LSF System <lsfadmin@hpc.dtu.dk>
Subject: Job 11047504: <myJob> in cluster <dcc> Done

Job <myJob> was submitted from host <gbarlogin2> by user <s202770> in cluster <dcc> at Tue Nov 16 12:39:04 2021
Job was executed on host(s) <n-62-20-13>, in queue <gpuv100>, as user <s202770> in cluster <dcc> at Tue Nov 16 12:39:07 2021
</zhome/24/8/154017> was used as the home directory.
</zhome/24/8/154017/TCR-pMHC-prediction/test> was used as the working directory.
Started at Tue Nov 16 12:39:07 2021
Terminated at Tue Nov 16 12:39:20 2021
Results reported at Tue Nov 16 12:39:20 2021

Your job looked like:

------------------------------------------------------------
# LSBATCH: User input
#!/bin/sh
 #BSUB -q gpuv100
 #BSUB -gpu "num=1"
 #BSUB -J myJob
 #BSUB -n 1
 #BSUB -W 01:00
 #BSUB -R "rusage[mem=32GB]"
 #BSUB -o logs/test.out
 #BSUB -e logs/test.err

 echo "Running script..."
 cd ~/TCR-pMHC-prediction/test
 python3 test.py

------------------------------------------------------------

Successfully completed.

Resource usage summary:

    CPU time :                                   1.94 sec.
    Max Memory :                                 112 MB
    Average Memory :                             112.00 MB
    Total Requested Memory :                     32768.00 MB
    Delta Memory :                               32656.00 MB
    Max Swap :                                   -
    Max Processes :                              4
    Max Threads :                                5
    Run time :                                   84 sec.
    Turnaround time :                            16 sec.

The output (if any) is above this job summary.



PS:

Read file <logs/test.err> for stderr output of this job.

